assets/sass/v2/_identify.scss

```diff
@@ -78,3 +78,19 @@
     }
   }
 }
+
+.o-form-button-bar {
+  .links-container {
+    font-size: $font-size-small;
+    margin-top: 1.9em;
+    margin-bottom: -1em;
+    max-width: 100%;
+    width: max-content;
```

Comment:
Check how very long label will show up in widget. I think it will not wrap. Should have to do this https://github.com/okta/okta-signin-widget/commit/be228d7742297d74e59317c4c6c22ea71ddfb314#diff-8f498508f801c71d47565b13055f8542f2b7bef687f82b4b308614caeacb825eR7

Reply:
Also the mouse pointer should change and be able to click only when it's hover over the link label.. rest of the container on the right side should not be clickable

Reply:
I've made the css change for the width. Even prior to this the wrapping was working but this is more appropriate so I still made the change. 

![image](https://user-images.githubusercontent.com/77295104/119558801-e2431a00-bd6f-11eb-897c-67efb7c2e12a.png)

Here's the demo for hover as well. It works as we expect:

https://user-images.githubusercontent.com/77295104/119558894-fdae2500-bd6f-11eb-824b-71132b0db513.mp4